### PR TITLE
libpano 13-2.9.21

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

previous effort, #92231

### CODE REVIEW

**Comment by chenrui333** [comment]
<details>
<summary>license ref (`version.h`)</summary>

```
/* Panorama_Tools	-	Generate, Edit and Convert Panoramic Images
   Copyright (C) 1998,1999 - Helmut Dersch  der@fh-furtwangen.de

   This program is free software; you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation; either version 2, or (at your option)
   any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License
   along with this software; see the file COPYING.  If not, a copy
   can be downloaded from http://www.gnu.org/licenses/gpl.html, or
   obtained by writing to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.  */

/*------------------------------------------------------------*/
```


</details>

File: Formula/libpano.rb [hunk 1]
```diff
@@ -20,20 +21,39 @@ class Libpano < Formula
...
   depends_on "libtiff"
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-  end
+  uses_from_macos "zlib"
+
+  patch :DATA
```
Comment (chenrui333) [inline Formula/libpano.rb:31]: gonna submit this build patch

Comment (chenrui333) [reply Formula/libpano.rb:31]: looks like this is raised by alebcay in the upstream.
https://groups.google.com/u/1/g/Hugin-ptx/c/FZrsgbz9SNo

**Comment by alebcay** [comment]
FYI I've opened discussion at https://groups.google.com/g/hugin-ptx/c/FZrsgbz9SNo for the header location issue, so far no responses (maybe I'm opening discussion at the wrong place).

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
The important thing is that it has been reported upstream. Let's ship this.

**Pull request closed without merging**